README.md

```diff
@@ -411,3 +438,20 @@ template.hbs:
 
 {{load-more-button action='infinityLoad' infinityModel=model}}
 ```
+
+### Using ember-infinity with non-blocking model hooks
+
+Chances are you might want to render your templates without blocking the render for the initial model hook.
+Ember-infinity works with this but there are some gotchas.
+
+Ensure your model returns a proper Ember-Data request, or use an Ember [PromiseProxy](https://www.emberjs.com/api/ember/2.14/classes/Ember.PromiseProxyMixin) so that ember knows how to render your promise once it is resolved.
```

Comment:
I'm not sure this note is necessary?  I'm sure 99% of users are using ember data, so this note feels a little confusing.

This feature has been designed to "just work", so I'd rather just imply here that it does just work, and simply that it didn't previously.

Something like:

```
In the world of optimistic route transitions & skeleton UI, it's necessary to return a POJO or similar primitive to Ember's `Route#model` hook to ensure the transition is not blocked by promise.

As of 1.0, the `infinityModel` hook now supports this behavior out of the box: